tests/components/matter/test_binary_sensor.py

```diff
@@ -344,3 +344,22 @@ async def test_water_valve(
     state = hass.states.get("binary_sensor.valve_valve_leaking")
     assert state
     assert state.state == "on"
+
+
+@pytest.mark.parametrize("node_fixture", ["thermostat"])
+async def test_thermostat_occupancy(
+    hass: HomeAssistant,
+    matter_client: MagicMock,
+    matter_node: MatterNode,
+) -> None:
+    """Test thermostat occupancy."""
+    state = hass.states.get("binary_sensor.longan_link_hvac_occupancy")
+    assert state
+    assert state.state == "on"
+
+    # Test Occupancy attribute change
+    set_node_attribute(matter_node, 1, 513, 2, 0)
```

Comment:
You can do without the hardcoded values here:

```suggestion
    occupany_attribute = clusters.Thermostat.Attributes.Occupancy
    
    set_node_attribute(
        matter_node, 
        1, 
        occupany_attribute.cluster_id, 
        occupany_attribute.attribute_id, 
        0
    )
```

Reply:
Fixed now